chat_exporter/construct/message.py

```diff
@@ -460,7 +479,8 @@ async def gather_messages(
             guild,
             meta_data,
             message_dict,
-        ).construct_message()
+            ).construct_message(asset_channel)
```

Comment:
Pass it in to `MessageConstruct` (as part of the class) than the `construct_message` function

Reply:
oh I forgot to remove that